.github/workflows/release.yaml

```diff
@@ -48,12 +48,12 @@ jobs:
         fetch-depth: 0
 
     - name: "Get tag name"
-      id: git_tag
-      run: |
-        echo ::set-output name=NAME::${GITHUB_REF/refs\/tags\//}
+      run: | # Trim "v" prefix in the release tag
+        RELEASE_TAG=${GITHUB_REF#refs/*/}
+        echo "BUILDERS_TAG=${RELEASE_TAG:1}" >> $GITHUB_ENV
```

Comment:
[opinion] I find unconditional `"${RELEASE_TAG:1}"` to be fragile.

I would prefer to add a check that `RELEASE_TAG` starts with `v` and fail pipeline otherwise.

Reply:
https://github.com/tetratelabs/getenvoy/pull/121/commits/cb1125ddf0b67c246295ce2ff7cdbba4a70003c5 done